sideboard/config.py

```diff
@@ -77,32 +166,33 @@ def get_config_files(requesting_file_path, plugin):
     order to facilitate testing modules which rely on Sideboard but which are
     not themselves Sideboard plugins.)
 
-    This function takes the following parameters:
-    requesting_file_path: the Python __file__ of the module which is parsing its
-                          config; used to locate development config files
-    plugin: boolean indicating whether config is being parsed for a plugin or
-            for Sideboard itself, since this affects which filenames we return
+    Parameters:
+        requesting_file_path (str): The Python __file__ of the module
+            requesting its config files.
+        is_plugin (bool): Indicates whether a plugin is making the request or
+            Sideboard itself is making the request, since this affects which
+            config files we return.
     """
     config_root = get_config_root()
-    module_dir, root_dir = get_dirnames(requesting_file_path, plugin)
+    module_dir, root_dir = get_module_and_root_dirs(requesting_file_path, is_plugin)
     module_name = os.path.basename(module_dir)
-    default_file_paths = ('development-defaults.ini', 'development.ini')
 
     if 'SIDEBOARD_MODULE_TESTING' in os.environ:
-        extra_configs = []
-    elif plugin:
-        extra_configs = [os.path.join(config_root, 'plugins.d', '{}.cfg'.format(module_name.replace('_', '-')))]
+        base_configs = []
+    elif is_plugin:
+        base_configs = [os.path.join(config_root, 'plugins.d', '{}.cfg'.format(module_name.replace('_', '-')))]
     else:
         assert module_name == 'sideboard', 'Unexpected module name {!r} requesting "non-plugin" configuration files'.format(module_name)
-        extra_configs = [
+        base_configs = [
             os.path.join(config_root, 'sideboard-core.cfg'),
-            os.path.join(config_root, 'sideboard-server.cfg'),
-        ]
+            os.path.join(config_root, 'sideboard-server.cfg')]
+
+    override_configs = [os.path.join(root_dir, config_path) for config_path in get_config_overrides()]
```

Comment:
Another really nitpicky thing: swapping the noun/adj here is confusing. That is, getting `override_configs` from a function called `get_config_overrides` makes the code slightly less readable.